components/label/Label.js

```diff
@@ -28,23 +28,38 @@ export default class Label extends PureComponent {
   };
 
   render() {
-    const { children, connectedLeft, connectedRight, className, inverse, helpText, required, size } = this.props;
+    const {
+      children,
+      connectedLeft,
+      connectedRight,
+      className,
+      inverse,
+      helpText,
+      required,
+      size,
+      ...others
+    } = this.props;
+
+    const boxProps = pickBoxProps({ ...others });
```

Comment:
you can just pass `others` instead of creating a new object and destructuring `others` into it.

Reply:
Also, I don't think you should pick the boxProps here. Now there might be props left out, not being passed to the Box anymore.